project.clj

```diff
@@ -88,7 +88,7 @@
                  [prismatic/schema "1.4.1"]
                  [metosin/schema-tools "0.13.1"]
                  [threatgrid/flanders "1.0.2"]
-                 [threatgrid/ctim "1.3.20"]
+                 [threatgrid/ctim "1.3.21-SNAPSHOT"]
```

Comment:
There is probably a typo. In pom.xml you have 1.3.21
```suggestion
                 [threatgrid/ctim "1.3.21"]
```

Reply:
@DeLaGuardo It's not ready for approval yet, I'm just trying to trigger the CI before I bump to the real CTIM release

Reply:
But I see that dependabot runs first so I have absolutely no way to test this without pulling 150MB worth of various docker containers...